sql/core/src/main/scala/org/apache/spark/sql/execution/joins/HashedRelation.scala

```diff
@@ -91,13 +96,18 @@ private[execution] object HashedRelation {
 
   /**
    * Create a HashedRelation from an Iterator of InternalRow.
+   *
+   * @param isLookupAware reserve one extra boolean in value to track if value being looked up
```

Comment:
Can you also mention that this is only used in full outer shuffled hash join?

Reply:
@viirya - sure, updated.